### DISCUSSION TITLE
Colobot minimun requirements

### DISCUSSION THREAD

**Discussion by ghost** [colobot/colobot#1637] [Q&A] [answered]
Hello everyone,

I'm interested in playing Colobot, but I'm having trouble finding information about the minimum requirements for the game. Can anyone here help me out?

If you've played Colobot or know where I can find this information, I would greatly appreciate it!

**Comment by masoudd** [comment]
Hello, the game is not at all resource intensive. I think if you can run a browser on your computer, then it'll be fine for colobot.

**Reply by ghost** [reply]
Ok, is because my computer is not so powerful, but ok 👍

**Comment by tomangelo2** [comment, marked as answer]
As long as your GPU supports OpenGL 3.3 (which should be supported by most GPUs released in last ~15 years or so) you should be fine. Current official build runs even with OpenGL 1.4 compatible GPUs (so about ~20 years old GPUs), but next update will raise that requirement.
For the CPU and RAM - I can't tell how old CPU must be to not be able to run the game. The game uses about <1GB of RAM (just quickly ran it, loaded a few levels and checked Task Manager), so there is a high chance your OS requires better hardware than the game.
Also it's not tested in Windows 8.1 and older, it's already known it won't run on Windows XP, some users reported issues with Windows 7. As long as the OS is actively supported (whether it's Windows, Linux or MacOS) it should work out of the box.

**Reply by ghost** [reply]
Ok, my CPU is not so powerful and i have only 4 GB of RAM (please, don't judge me), my OS is Linux. I played the game for some hours (the game is very funny) and the game worked well on my computer.
